packages/js/components/src/experimental-tree-control/tree-item.scss

```diff
@@ -1,3 +1,5 @@
+@import 'node_modules/@wordpress/base-styles/colors.native';
```

Comment:
I needed to do this for `$gray-0` to be defined. Is this the best way of doing this?

Reply:
https://github.com/ROBERLANDIO/-octaedro-approved-this-pull-request.-Nice-job-joshuatf-This-is-testing-well-and-the-code-

Reply:
I think this is fine.  We might consider moving it to the top of `packages/js/components/src/style.scss` so it can be shared with other components in the future.